tests/fixtures/NoBasicAssignmentFunctionParameterLinter/function_basic_assignment.php.in

```diff
@@ -0,0 +1,18 @@
+<?hh // strict
+/**
+ * Copyright (c) 2016, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
These should be the MIT license as shown in the real source files - please fix them in this change - I'll fix all the existing ones in a separate commit.